.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: scala
+
+scala:
+    - 2.13.1
```

Comment:
The four spaces do not look like yaml to me

Reply:
And doesn’t this work without being an array?

Reply:
https://docs.travis-ci.com/user/languages/scala/#specifying-scala-versions